CONTRIBUTING.md

```diff
@@ -199,63 +202,65 @@ When you're done, you can restore your Redwood App to its original state by dele
 
 You can avoid having to provide the path to `redwood` by defining an `RW_PATH` environment variable on your system.
 
-_On Linux_
+  ##### On **Linux**
```

Comment:
Making these actual sections is an improvement, but the formatting seems a little off; it might be because of the indentation? I don't think we need all of these sections in code blocks, just the code parts

![image](https://user-images.githubusercontent.com/32992335/119233762-55822d00-badf-11eb-8ae3-5ad85f16b38c.png)

Reply:
my fault. should be fixed in latest iteration. please confirm.